src/definitions/behaviors/form.js

```diff
@@ -1349,13 +1349,14 @@
                             fieldErrors = [],
                             isDisabled = $field.filter(':not(:disabled)').length === 0,
                             validationMessage = $field[0].validationMessage,
+                            noNativeValidation = $field.filter("[native-validation='false']").length === 1,
```

Comment:
1. As the default is to always use native validation, an explicit setting to "false" is IMHO not needed.
Instead, we should just check for an existing "no-native-validation" property
2. I also suggest to support a global setting so a whole form can ignore native validation for each field
3. We should also support `noNativeValidation: true` as a field option when given via field-rules on instantiation

```suggestion
                            noNativeValidation = $field.filter("[no-native-validation]").length > 0 || field.noNativeValidation || settings.noNativeValidation,
```

Please also add `noNativeValidation: false` to the settings object (i suggest after `errorLimit` https://github.com/fomantic/Fomantic-UI/blob/5eb2cfde0c188fb48e504481f0d6be333e8859ff/src/definitions/behaviors/form.js#L1630C23-L1630C23)

Reply:
I did some more investigation and i think we need to enhance this even more making use ot the standard novalidate form property and formnovalidate submit property (which we could extend to each input field) i'll update my change request above accordingly